### PR TITLE
chore: update to version 0.9.5

### PR DESCRIPTION
**Pull request by serena-ruan**

### CODE REVIEW

**Comment by serena-ruan** [comment]
Waiting for:
1. merging this https://github.com/microsoft/SynapseML/pull/1245 for website enhancement -- then update the versions in website folder altogether
2. publishing speech jar to maven central

File: website/versioned_docs/version-0.9.5/reference/cyber.md [hunk 1]
```diff
@@ -18,65 +18,65 @@ sidebar_label: CyberML
    (i.e., it returns a sample from the complement set).
 
 ## feature engineering: [indexers.py](https://github.com/microsoft/SynapseML/blob/master/core/src/main/python/synapse/ml/cyber/feature/indexers.py)
-1. [IdIndexer](https://mmlspark.blob.core.windows.net/docs/0.9.4/pyspark/synapse.ml.cyber.feature.html#synapse.ml.cyber.feature.indexers.IdIndexer)
+1. [IdIndexer](https://mmlspark.blob.core.windows.net/docs/0.9.5/pyspark/synapse.ml.cyber.feature.html#synapse.ml.cyber.feature.indexers.IdIndexer)
```
Comment (imatiach-msft) [inline website/versioned_docs/version-0.9.5/reference/cyber.md:21]: maybe we should just have a "latest" tag here in the future?  Changing all of these links is error prone on every version update.

**Review by imatiach-msft — APPROVED** [review]

File: website/versioned_docs/version-0.9.5/reference/cyber.md (hunk 1 above)
Comment (mhamilton723) [reply website/versioned_docs/version-0.9.5/reference/cyber.md:21]: This is a good idea and we will investigate how to do this

Comment (serena-ruan) [reply website/versioned_docs/version-0.9.5/reference/cyber.md:21]: We'll just update the `next` version, and all previous staged version won't be changed then.

File: notebooks/features/cognitive_services/CognitiveServices - Overview.ipynb [hunk 2]
```diff
@@ -30,66 +30,66 @@
...
+    "- Named entity recognition: identifies known entities and general named entities in the input text ([Scala](https://mmlspark.blob.core.windows.net/docs/0.9.5/scala/com/microsoft/azure/synapse/ml/cognitive/NER.html), [Python](https://mmlspark.blob.core.windows.net/docs/0.9.5/pyspark/synapse.ml.cognitive.html#module-synapse.ml.cognitive.NER))\n",
+    "- Sentiment analysis: returns a score betwee 0 and 1 indicating the sentiment in the input text ([Scala](https://mmlspark.blob.core.windows.net/docs/0.9.5/scala/com/microsoft/azure/synapse/ml/cognitive/TextSentiment.html), [Python](https://mmlspark.blob.core.windows.net/docs/0.9.5/pyspark/synapse.ml.cognitive.html#module-synapse.ml.cognitive.TextSentiment))\n",
+    "- Healthcare Entity Extraction: Extracts medical entities and relationships from text. ([Scala](https://mmlspark.blob.core.windows.net/docs/0.9.5-41-9e51ba7d-SNAPSHOT/scala/com/microsoft/azure/synapse/ml/cognitive/HealthcareSDK.html), [Python](https://mmlspark.blob.core.windows.net/docs/0.9.5-41-9e51ba7d-SNAPSHOT/pyspark/synapse.ml.cognitive.html#module-synapse.ml.cognitive.HealthcareSDK))\n",
```
Comment (mhamilton723) [inline notebooks/features/cognitive_services/CognitiveServices - Overview.ipynb:61]: ``
```suggestion
    "- Healthcare Entity Extraction: Extracts medical entities and relationships from text. ([Scala](https://mmlspark.blob.core.windows.net/docs/0.9.5/scala/com/microsoft/azure/synapse/ml/cognitive/HealthcareSDK.html), [Python](https://mmlspark.blob.core.windows.net/docs/0.9.5/pyspark/synapse.ml.cognitive.html#module-synapse.ml.cognitive.HealthcareSDK))\n",
```

File: notebooks/features/cognitive_services/CognitiveServices - Overview.ipynb [hunk 3]
```diff
@@ -30,66 +30,66 @@
...
+    "- Speech-to-text: transcribes audio streams ([Scala](https://mmlspark.blob.core.windows.net/docs/0.9.5/scala/com/microsoft/azure/synapse/ml/cognitive/SpeechToText.html), [Python](https://mmlspark.blob.core.windows.net/docs/0.9.5/pyspark/synapse.ml.cognitive.html#module-synapse.ml.cognitive.SpeechToText))\n",
+    "- Conversation Transcription: transcribes audio streams into live transcripts with identified speakers. ([Scala](https://mmlspark.blob.core.windows.net/docs/0.9.5/scala/com/microsoft/azure/synapse/ml/cognitive/ConversationTranscription.html), [Python](https://mmlspark.blob.core.windows.net/docs/0.9.5/pyspark/synapse.ml.cognitive.html#module-synapse.ml.cognitive.ConversationTranscription))\n",
+    "- Text to Speech: Converts text to realistic audio ([Scala](https://mmlspark.blob.core.windows.net/docs/0.9.5-41-9e51ba7d-SNAPSHOT/scala/com/microsoft/azure/synapse/ml/cognitive/TextToSpeech.html), [Python](https://mmlspark.blob.core.windows.net/docs/0.9.5-41-9e51ba7d-SNAPSHOT/pyspark/synapse.ml.cognitive.html#module-synapse.ml.cognitive.TextToSpeech))\n",
```
Comment (mhamilton723) [inline notebooks/features/cognitive_services/CognitiveServices - Overview.ipynb:52]:
```suggestion
    "- Text to Speech: Converts text to realistic audio ([Scala](https://mmlspark.blob.core.windows.net/docs/0.9.5/scala/com/microsoft/azure/synapse/ml/cognitive/TextToSpeech.html), [Python](https://mmlspark.blob.core.windows.net/docs/0.9.5/pyspark/synapse.ml.cognitive.html#module-synapse.ml.cognitive.TextToSpeech))\n",
```

File: website/docs/getting_started/installation.md [hunk 4]
```diff
@@ -73,7 +73,7 @@ Excluding certain packages from the library may be necessary due to current issu
 {
     "name": "synapseml",
     "conf": {
-        "spark.jars.packages": "com.microsoft.azure:synapseml_2.12:0.9.4",
+        "spark.jars.packages": "com.microsoft.azure:synapseml_2.12:0.9.5",
         "spark.jars.repositories": "https://mmlspark.azureedge.net/maven",
```
Comment (mhamilton723) [inline website/docs/getting_started/installation.md:77]:
```suggestion
```

File: website/docs/getting_started/installation.md [hunk 5]
```diff
@@ -87,7 +87,7 @@ In Azure Synapse, "spark.yarn.user.classpath.first" should be set to "true" to o
...
```
Comment (mhamilton723) [inline website/docs/getting_started/installation.md:91]:
```suggestion
```

File: website/src/pages/index.js [hunk 6]
```diff
@@ -373,7 +373,7 @@ spark-submit --packages com.microsoft.azure:synapseml_2.12:0.9.4 MyApp.jar --con
                   <CodeSnippet
                     snippet={`import pyspark
 spark = pyspark.sql.SparkSession.builder.appName("MyApp")
-        .config("spark.jars.packages", "com.microsoft.azure:synapseml_2.12:0.9.4")
+        .config("spark.jars.packages", "com.microsoft.azure:synapseml_2.12:0.9.5")
         .config("spark.jars.repositories", "https://mmlspark.azureedge.net/maven")
```
Comment (mhamilton723) [inline website/src/pages/index.js:377]:
```suggestion
```

File: website/src/pages/index.js [hunk 7]
```diff
@@ -288,9 +288,9 @@ function Home() {
                   SynapseML can be conveniently installed on existing Spark
                   clusters via the --packages option, examples:
                   <CodeSnippet
-                    snippet={`spark-shell --packages com.microsoft.azure:synapseml_2.12:0.9.4 --conf spark.jars.repositories=https://mmlspark.azureedge.net/maven
-pyspark --packages com.microsoft.azure:synapseml_2.12:0.9.4 --conf spark.jars.repositories=https://mmlspark.azureedge.net/maven
-spark-submit --packages com.microsoft.azure:synapseml_2.12:0.9.4 MyApp.jar --conf spark.jars.repositories=https://mmlspark.azureedge.net/maven`}
+                    snippet={`spark-shell --packages com.microsoft.azure:synapseml_2.12:0.9.5 --conf spark.jars.repositories=https://mmlspark.azureedge.net/maven
+pyspark --packages com.microsoft.azure:synapseml_2.12:0.9.5 --conf spark.jars.repositories=https://mmlspark.azureedge.net/maven
+spark-submit --packages com.microsoft.azure:synapseml_2.12:0.9.5 MyApp.jar --conf spark.jars.repositories=https://mmlspark.azureedge.net/maven`}
```
Comment (mhamilton723) [inline website/src/pages/index.js:293]:
```suggestion
                    snippet={`spark-shell --packages com.microsoft.azure:synapseml_2.12:0.9.5
pyspark --packages com.microsoft.azure:synapseml_2.12:0.9.5
spark-submit --packages com.microsoft.azure:synapseml_2.12:0.9.5 MyApp.jar `}
```

**Pull request merged**